broker/models/Location.js

```diff
@@ -0,0 +1,66 @@
+/*
+Location {
+  status : enum (acitve, suspended, inactive)
```

Comment:
You need to add an identifier element so we can store the WIR client ID in this resource: https://www.hl7.org/fhir/location-definitions.html#Location.identifier